tidb/manifest.json

```
@@ -29,8 +28,8 @@
       "spec": "assets/configuration/spec.yaml"
     },
     "dashboards": {
-      "overview": "assets/dashboards/one-screen-overview.json",
-      "TiDB Overview": "assets/dashboards/overview.json"
+      "TiDB Overview": "assets/dashboards/overview.json",
+      "overview": "assets/dashboards/overview.json"
```

```suggestion
      "TiDB Overview": "assets/dashboards/overview.json"
```
After that you can delete the dashboard file